plugins/GSdx/GSdx.cpp

```diff
@@ -155,12 +155,14 @@ GSdxApp::GSdxApp()
 	m_gs_aspectratio.push_back(GSSetting(1, "4:3", ""));
 	m_gs_aspectratio.push_back(GSSetting(2, "16:9", ""));
 
-	m_gs_upscale_multiplier.push_back(GSSetting(1, "Custom", ""));
+	m_gs_upscale_multiplier.push_back(GSSetting(1, "Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(2, "2x Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(3, "3x Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(4, "4x Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(5, "5x Native", ""));
 	m_gs_upscale_multiplier.push_back(GSSetting(6, "6x Native", ""));
+	m_gs_upscale_multiplier.push_back(GSSetting(8, "8x Native", ""));
+	m_gs_upscale_multiplier.push_back(GSSetting(9, "Custom", ""));
```

Comment:
Yes. As a side note, it would have been better to put custom resolution in position 0. Anyway, keep 9 as custom will be removed anyway.

However all calls to ``` theApp.GetConfig("upscale_multiplier",1) ``` must use 1 as default value. Native must be the default value.